azure_pipelines/pr-validation.yml

```diff
@@ -83,4 +83,62 @@ jobs:
       testResultsFiles: '$(Agent.BuildDirectory)/s/build/reports/*'
       failTaskOnFailedTests: true
       testRunTitle: 'Test Run - $(target)'
-  
+
+- job: 'Validate_SPM_Integration'
+  displayName: Validate SPM Integration
+  pool:
+    vmImage: 'macOS-13'
+    timeOutInMinutes: 15
+  workspace:
+    clean: all
+
+  steps:
+
+  - checkout: self
+    clean: true
+    submodules: true
+    fetchDepth: 1
+    persistCredentials: true
```

Comment:
Should this be false?

Reply:
I tried, but got this [error](https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=1320612&view=logs&j=1494a728-882c-596b-7b5c-9fa51f14f0ef&t=7be9455c-7140-52f8-03cd-b09f3cf00ddb): `fatal: could not read Username for 'https://github.com'/: terminal prompts disabled`

The [documentation](https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-checkout?view=azure-pipelines) says that it is necessary in order to leave the OAuth token in the Git config after the initial fetch.